src/modules/SdnDiag.Health/private/Test-NcUrlNameResolution.ps1

```diff
@@ -21,7 +21,7 @@ function Test-NcUrlNameResolution {
     try {
         "Validate that the Network Controller NB API URL resolves to the correct IP address" | Trace-Output
 
-        $ncApiReplicaPrimary = Get-SdnServiceFabricReplica -NetworkController $SdnEnvironmentObject.ComputerName -Credential $Credential -ServiceTypeName 'ApiService' -Primary
+        $ncApiReplicaPrimary = Get-SdnServiceFabricReplica -NetworkController $SdnEnvironmentObject.ComputerName[0] -Credential $Credential -ServiceTypeName 'ApiService' -Primary
```

Comment:
There are two types of NetworKController parameter assignment
`-NetworkController $SdnEnvironmentObject.ComputerName[0]`
and `-NetworkController $SdnEnvironmentObject.EnvironmentInfo.NetworkController[0]`. What will be the difference here and if make sense we may make it consistent.